commit ce1333961cf9e1ceb3632b31e4b008f2b6174ab9
Author: bertin <macportsuser@macports.org>
Date:   Fri May 23 21:30:16 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts-pextlib/files/lamarck/patch-pextlib-no-sip.diff

diff --git a/src/pextlib1.0/sip_copy_proc.c b/src/pextlib1.0/sip_copy_proc.c
index 10aa5df8..994c625a 100644
--- a/src/pextlib1.0/sip_copy_proc.c
+++ b/src/pextlib1.0/sip_copy_proc.c
@@ -335,6 +335,11 @@ resign_out:
 }
 
 static char *lazy_copy(const char *path, struct stat *in_st) {
+#ifndef SF_RESTRICTED
+    (void) path;
+    (void) in_st;
+    return NULL;
+#else
     char *retval = NULL;
     uid_t euid = geteuid();
     int outfd = -1;
@@ -529,6 +534,7 @@ lazy_copy_out:
         free(target_path);
     }
     return retval;
+#endif // SF_RESTRICTED
 }
 
 /**
